src/poke_env/environment/effect.py

```diff
@@ -243,6 +265,31 @@ def from_showdown_message(message: str) -> Effect:
             )
             return Effect.UNKNOWN
 
+    @staticmethod
+    def from_data(message: str) -> Effect:
+        """Returns the Effect object corresponding to the string in static data.
+
+        :param message: The message to convert.
+        :type message: str
+        :return: The corresponding Effect object.
+        :rtype: Effect
+        """
+        message = message.replace("_", "")
+        message = message.replace(" ", "")
+        message = message.replace("-", "")
+        message = message.upper()
+        try:
+            return _FROM_DATA[message]
+        except KeyError:
+            logging.getLogger("poke-env").warning(
```

Comment:
Can you add a test for this path? Eg. calling `from_data` on a non-existing effect